documentation/src/main/asciidoc/topics/contributing/terminology.adoc

```diff
@@ -0,0 +1,81 @@
+[[terminology]]
+= {brandname} Terminology
+
+{brandname} terminology refers to words and phrases specific to {brandname}
+capabilities and features. It is important to use consistent terminology across
+documentation and the code base to avoid unclear or contradictory language.
+
+Add::
+Add an item to an existing list, group, view, or other container element. For example, "add values to counters" or "add entries to caches".
+
+Cache configuration::
+A file in XML or JSON format that controls {brandname} behavior.
+
+Cache template::
+An artifact from which you create cache configurations. +
+Templates can be pre-defined cache configurations such as in the `org.infinispan.client.hotrod.DefaultTemplate` enum. +
+Users can use existing cache configurations as templates. +
+Users can also define templates in `infinispan.xml` with the `*-cache-configuration` tags.
+
+Cache definition::
+Do not use to mean "cache configuration".
```

Comment:
This is one of the things that I intend on changing in the docs. I've noticed all the instances of "cache definition" too. I'm also guilty of using that phrase. It would be better to be consistent and say "cache configuration" instead. I think it would be fine to say "define a cache configuration" but I don't think we need two ways to say the one thing. 

I'll update the terminology docs to add definitions for these two.